src/ringcentral-web-phone.js

```diff
@@ -206,6 +206,12 @@
             this.userAgent.audioHelper.playIncoming(true);
             patchSession(session);
             patchIncomingSession(session);
+            session._sendReceiveConfirmPromise = session.sendReceiveConfirm().then(function() {
+                session.logger.log('sendReceiveConfirm success');
+            }).catch(function(error){
+                session.logger.error('failed to send receive confirmation via SIP MESSAGE due to ' + error);
+                return Promise.reject(error);
```

Comment:
Throw the error again:

```js
throw error;
```

Reply:
`throw error` is equal as `return Promise.reject(error)`.   If you prefer it, I will update.

Reply:
Yes, rethrow is more appropriate